src/box/lua/feedback_daemon.lua

```diff
@@ -26,13 +26,105 @@ local function get_fiber_id(f)
     return fid
 end
 
-local function fill_in_feedback(feedback)
+local function detect_docker_environment()
+    local fh = fio.open('/proc/1/cgroup', {'O_RDONLY'})
+    if not fh then return false end
+
+    -- fh:read() doesn't read empty files
+    local big_enough_chunk = 4096
+    local ok = fh:read(big_enough_chunk)
+    fh:close()
+    if not ok then return false end
+
+    if not string.find(ok, 'docker') then return false end
+
+    return true
```

Comment:
`return str and str:find('docker') or false`?

Reply:
thanks, actually it's:

```lua
return str and str:find('docker') and true or false
```